lib/avo/resources/controls/action.rb

```diff
@@ -4,17 +4,17 @@ module Controls
       class Action < BaseControl
         attr_reader :klass
 
-        def initialize(klass, model: nil, resource: nil, view: nil, **args)
+        def initialize(klass, record: nil, resource: nil, view: nil, **args)
           super(**args)
 
           @klass = klass
           @resource = resource
-          @model = model
+          @record = record
           @view = view
         end
 
         def action
-          @instance ||= @klass.new(model: @model, resource: @resource, view: @view)
+          @instance ||= @klass.new(model: @record, resource: @resource, view: @view)
```

Comment:
For some reason this class is initialized by `Avo::Resources::Controls::Action.new(klass, record: record, resource: resource, view: view, **args)` but the `initialize` method was not expecting the same argument name, using `model` instead `record`. (In avo3 this is already uniformed)